userspace/libscap/scap_open.h

```diff
@@ -86,6 +86,8 @@ typedef struct scap_open_args
 	// values via scap_suppress_events_comm().
 	bool udig; ///< If true, UDIG will be used for event capture.
 	const char *gvisor_socket; ///< If not null, gvisor will be used for event capture.
+	const char *gvisor_root_path; ///< When using gvisor, the root path used by runsc commands
+	const char *gvisor_trace_session_path; ///< When using gvisor, the trace session config file
```

Comment:
I think it might be worth to create a struct around these; yet i have no strong opinions on doing that now.

Reply:
Also, we might want to tweak them a little when adding this to Falco. I'm ok with that but I understand your comment.

Reply:
I think we can refactor this when we are sure of all the things that are actually needed. For instance, in the next PR the `gvisor_socket` may be removed from here, because it can be retrieved from the config we will take as input, so my advice is to wait until we have a more precise shape